content/blog/python-for-pac/index.md

```diff
@@ -0,0 +1,190 @@
+                "Read more about read access here: " +
+                "https://docs.microsoft.com/en-us/azure/storage/blobs/storage-manage-access-to-resources")
+
+storage_container_no_public_read = ResourceValidationPolicy(
+    name="storage-container-no-public-read",
+    description="Prohibits setting the public permission on Azure Storage Blob Containers.",
+    validate=storage_container_no_public_read_validator,
+)
+
+PolicyPack(
+    name="azure-python",
+    enforcement_level=EnforcementLevel.MANDATORY,
+    policies=[
+        storage_container_no_public_read,
+    ],
+)
+```
+
+{{% /choosable %}}
+{{% choosable cloud gcp %}}
+
+```python
+from pulumi_policy import (
+    EnforcementLevel,
+    PolicyPack,
+    ReportViolation,
+    ResourceValidationArgs,
+    ResourceValidationPolicy,
+)
+
+def storage_bucket_no_public_read_validator(args: ResourceValidationArgs, report_violation: ReportViolation):
+    if args.resource_type == "gcp:storage/bucketACL:BucketACL" and "predefinedAcl" in args.props:
+        acl = args.props["predefinedAcl"]
+        if acl == "public-read" or acl == "public-read-write":
+            report_violation("Storage buckets acl cannot be set to public-read or public-read-write.")
+
+storage_bucket_no_public_read = ResourceValidationPolicy(
+    name="storage-bucket-no-public-read",
+    description="Prohibits setting the publicRead or publicReadWrite permission on GCP Storage buckets.",
+    validate=storage_bucket_no_public_read_validator,
+)
+
+PolicyPack(
+    name="gcp-python",
+    enforcement_level=EnforcementLevel.MANDATORY,
+    policies=[
+        storage_bucket_no_public_read,
+    ],
+)
+```
+
+{{% /choosable %}}
+{{< /chooser >}}
+
+In the case of this Kubernetes policy example, the policy does not allow deploying a loadbalancer which would open the kubernetes cluster to the public Internet.
+
+```python
+from pulumi_policy import (
+    EnforcementLevel,
+    PolicyPack,
+    ReportViolation,
+    ResourceValidationArgs,
+    ResourceValidationPolicy,
+)
+
+def no_public_services_validator(args: ResourceValidationArgs, report_violation: ReportViolation):
+    if args.resource_type == "kubernetes:core/v1:Service" and "spec" in args.props:
+        spec = args.props["spec"]
+        if "type" in spec and spec["type"] == "LoadBalancer":
+            report_violation(
+                "Kubernetes Services cannot be of type LoadBalancer, which are exposed to " +
+                "anything that can reach the Kubernetes cluster. This likely including the " +
+                "public Internet.")
+
+no_public_services = ResourceValidationPolicy(
+    name="no-public-services",
+    description="Kubernetes Services should be cluster-private.",
+    validate=no_public_services_validator,
+)
+
+PolicyPack(
+    name="kubernetes-python",
+    enforcement_level=EnforcementLevel.MANDATORY,
+    policies=[
+        no_public_services,
+    ],
+)
+```
+
+## Summary
```

Comment:
Should we add a couple sentences about how to run such a policy pack locally, before the summary? Something like:

> If you'd like to try out one of the examples above, you can create a policy with `pulumi policy new` and to run locally against a Pulumi program specify `pulumi up --policy-pack <path-to-policy-pack>`?

Also wondering if we should mention that you can enforce policy packs in your organization by publishing the policy pack (running `pulumi policy publish [org-name]` inside the policy pack dir). That's what's new in GA, the ability to publish and enforce. Though, that's limited to customers that have support for this enabled in the Pulumi Console. So I could also see leaving it off.

Reply:
I'll add the suggestion to try out the policy but leave out publishing in an org